changelog/10525.bugfix.rst

```diff
@@ -0,0 +1 @@
+Make classmethods discoverable. Same logic as staticmethods.
```

Comment:
I think this file should probably be a "feature".

```suggestion
Test methods decorated with ``@classmethod`` can now be discovered as tests, following the same rules as normal methods. This fills the gap that ``@staticmethod``s were discoverable as tests but not ``@classmethod``s.
```

I think the above wording yields more context about the change.